src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @java.lang.Override
+    public void doWork() {
+        System.out.println("Bulldozer started it's work");
+    }
+
+    @java.lang.Override
```

Comment:
```suggestion
    @Override
```

Fix it in all places